src/main/java/assets/javascript/blackjack.js

```diff
@@ -20,6 +20,41 @@ function displayMoney(game){
     }
 }
 
+function displayUserCards(game){
+    if (game.needBet == true){
+        alert ("You must bet a minimum of $2");
+    }else{
+        //Rebuild user hand now
+        //First remove all user card divs from view
+        $("#userCards").html("");
+        //Iteratively add users cards from game.pHand
+        for(i=0;i<game.pHand.length;i++){
+            card = game.pHand[i];
+            cardDiv = "<div><img src='/assets/cards/" + card.value + card.suit.charAt(0).toLowerCase() + ".png'></div>";
+            $("#userCards").append(cardDiv);
+        }
+    }
+}
+
+$("#deal").click(function(){
+    state = $("#deal").text();
+    if (game.stillBet == true){
```

Comment:
This should check for false. In your code, stillBet is equal to true while they are still betting and set to false when they are done betting. It will always deal them cards because stillBet was initialized to true and tryDeal() was never called (even if bets are less than 2). You need to do ajax first, call tryDeal, then in the success portion check if the deal was successful